rust/tvm-macros/src/external.rs

```diff
@@ -17,12 +17,32 @@
  * under the License.
  */
 use proc_macro2::Span;
+use proc_macro_error::abort;
 use quote::quote;
 use syn::parse::{Parse, ParseStream, Result};
 
-use syn::{FnArg, Generics, Ident, Lit, Meta, NestedMeta, Pat, ReturnType, TraitItemMethod, Type};
+use syn::{FnArg, Signature, Attribute, token::Semi, Visibility, Generics, Ident, Lit, Meta, NestedMeta, Pat, ReturnType, Type};
+
+struct ExternalItem {
```

Comment:
nit: brief rustdoc? Some comments would help me get the story of what `ExternalItem` and `External` are for.

Reply:
What Greg said